src/diffusers/models/autoencoders/autoencoder_kl_cogvideox.py

```diff
@@ -950,25 +954,65 @@ def __init__(
         self.use_slicing = False
         self.use_tiling = False
 
-        self.tile_sample_min_size = self.config.sample_size
-        sample_size = (
-            self.config.sample_size[0]
-            if isinstance(self.config.sample_size, (list, tuple))
-            else self.config.sample_size
+        # We make the minimum height and width of sample for tiling half that of the generally supported
+        self.tile_sample_min_height = sample_height // 2
+        self.tile_sample_min_width = sample_width // 2
+        self.tile_latent_min_height = int(
+            self.tile_sample_min_height / (2 ** (len(self.config.block_out_channels) - 1))
         )
-        self.tile_latent_min_size = int(sample_size / (2 ** (len(self.config.block_out_channels) - 1)))
-        self.tile_overlap_factor = 0.25
+        self.tile_latent_min_width = int(self.tile_sample_min_width / (2 ** (len(self.config.block_out_channels) - 1)))
+        self.tile_overlap_factor = 0.33
```

Comment:
How was this determined?

Reply:
Mostly trial and error of what seemed to give the least artifacting with my rough implementation earlier. I think it can be lowered now that I have cleaned up the implementation and made it similar to AutoencoderKL. It needs to be chosen such that the latent height/width (60/90) are divisible nicely

Reply:
Let's add this as a comment then.

Reply:
Turns out 1/6 for height and 1/5 for width allow for near-perfect visual similarity (even better than the examples in the PR description) without the tiling effect showing up. I've added an explanation that these were experimentally chosen and only tested on the strongly recommended generation resolutions of 720x480 for CogVideoX-2b. In the future, we can revisit this when the newer and upcoming version of CogVideo support multi-resolution high quality generation. Shouldn't be too hard to make it resolution agnostic eventually

Reply:
Thanks for looking into it! Should we then try make it configurable as it seems to be a bit of hit and miss?

Reply:
As parameters in enable_tiling? Sure, works